### PR TITLE
Update bug report template

### PR DESCRIPTION
**Pull request by nyechiel**
Updating the Issue template now that `subctl gather` and `subctl diagnose` are available for users. See also submariner-io/submariner#1304

Signed-off-by: nyechiel <nyechiel@redhat.com>

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

**Comment by nyechiel** [comment]
The CI failure is not related this this change.

**Comment by tpantelis** [comment]
> The CI failure is not related this this change.

yes - we can override the failure and merge - just waiting for one more approval.

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**
